src/lib/constants.py

```
@@ -104,8 +104,8 @@
 QUALITY_TEST_PROVIDERS = [
     #"gpt-5.4",
     "gpt-5.4-mini",
-    "gpt-5.4-nano",
-    "gpt-5.3-chat-latest",
+    #"gpt-5.4-nano",
```

Flake8 (7.3.0)

[error] block comment should start with '# '

(E265)